wat

```diff
@@ -22,6 +26,13 @@ mark_lines() {
 
 : ${PACMAN_LOG:=/var/log/pacman.log}
 
+while [ $# -gt 0 ]; do
+    case $1 in
+        -i|--installed) SHOW_INSTALLED=1; shift ;;
```

Comment:
Yup, added in 45cad3943bd821d1e3e2653cce2f7da11e2a8932

On Mon, Feb 2, 2015 at 3:40 PM, pat brisbin <notifications@github.com>
wrote:

> In wat <https://github.com/pbrisbin/wat/pull/2#discussion_r23956408>:
>
> > @@ -22,6 +26,13 @@ mark_lines() {
> >
> >  : ${PACMAN_LOG:=/var/log/pacman.log}
> >
> > +while [ $# -gt 0 ]; do
> > +    case $1 in
> > +        -i|--installed) SHOW_INSTALLED=1; shift ;;
>
> That seems reasonable. I would initialize it to 0 to make that explicit,
> up with and following the same syntax as PACMAN_LOG.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/pbrisbin/wat/pull/2/files#r23956408>.
>